tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,164 @@
+              <param name="dataproduct_type" type="text" label="Data product type" help="Logical data product type (image etc.)" />
+              <param name="obs_collection" type="text" label="Collection" help="Name of the data collection" />
+              <param name="facility_name" type="text" label="Facility name" help="Name of the facility used for this observation" />
+              <param name="instrument_name" type="text" label="Instrument name" help="Name of the instrument used for this observation" />
+              <param name="em_min" type="text" label="Spectral coordinates (start)" />
+              <param name="em_max" type="text" label="Spectral coordinates (end)" />
+              <param name="target_name" type="text" label="Target name" help="Name of the astronomical object observed" />
+              <param name="obs_publisher_id" type="text" label="Publisher dataset ID" />
+              <param name="s_fov" type="text" label="Bounds of the covered region" />
+              <param name="calibration_level" type="select" label="Calibration level (0, 1, 2, 3, 4, 5)" >
+                <option value="none">None</option>
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+              </param>
+              <param name="order_by" type="select" label="Order by" help="Field to use for ordering the results">
+                <option value="none" selected="true">No specific order</option>
+                <option value="size" >Order by size</option>
+                <option value="collection" >Order by collection</option>
+                <option value="object" >Order by name</option>
+              </param>
+            </when>
+            <when value="raw_query">
+              <param name="table" type="text" label="Table name" help="Name of the table you want to query (FROM clause)" required="True" />
+              <section name="where_clause" title="WHERE clause (field = value)" expanded="false">
+                <param name="where_field" type="text" label="WHERE field" help="WHERE **field** = value" />
+                <param name="where_condition" type="text" label="WHERE value" help="WHERE field = **value**" />
+              </section>
+              <param name="url_field" type="text" label="Url field" help="Table field containing the url of the file to download" required="True" />
+            </when>
+          </conditional>
+        </section>
+    </inputs>
+    <outputs>
+
+
+        <data name="output" format="txt" label="${tool.name} -> ${archiveselection.archive_type}:" >
+          <discover_datasets pattern="__designation_and_ext__" directory="fits" visible="true" format="fits" />
+        </data>
+
+    </outputs>
+    <help><![CDATA[
+Archives description :
```

Comment:
just for the record, the help does not look right in galaxy. It is however a  priority at a later time.

![image](https://github.com/esg-epfl-apc/tools-astro/assets/3909535/2dde92df-47f1-4aca-8fe3-8207e81ab9f2)